### DISCUSSION TITLE
notifications

### DISCUSSION THREAD

**Discussion by anauk** [invertase/react-native-firebase#8058] [Q&A]
Hello! I stuck with issue. Now i don’t get notifications in my react-native app. When i check @react-native-firebase/messaging:  ^21.0.0 i don't see onMessage() method.
<img width="1631" alt="Screenshot 2024-10-11 at 10 44 34 AM" src="https://github.com/user-attachments/assets/ea6610cd-0824-488f-8a23-3fef9b6757da">
What I'm doing wrong?

**Comment by mikehardy** [comment]
Hi there -

1- whether you see "onMessage" on the native module or not is not a valid test for whether functionality exists
2- it's there at the javascript layer, and if you use the github search tool for "onMessage" you'll see it

If you are reporting an issue, it's best to actually report an issue and fill out the template + include all the versions that are in use + include code that reproduces etc

But in general `onMessage` / not receiving messages issues are related to local project issues and a careful careful careful examination of the *device logs* (adb logcat / or Console.app looking at logs of a real iOS device plugged into computer) will show that the FCM are delivered fine but the app isn't receiving them for some reason.

**Comment by anauk** [comment]
Yes, it is true. Thank you for your answer

пт, 11 окт. 2024 г. в 14:13, Mike Hardy ***@***.***>:
